src/fixup.js

```diff
@@ -297,6 +297,26 @@
     request.send();
   }
 
+  /* Dark mode toggle */
+  const darkCss = document.querySelector('link[href="dark.css"]');
+  if (darkCss) {
+    const btn = document.createElement('button');
+    btn.id = 'theme-toggle';
+    btn.innerText = "Toggle dark mode";
+    document.body.appendChild(btn);
+
+    const currentTheme = localStorage.getItem("tr-theme");
+    if (currentTheme === "light") {
+      darkCss.disabled = true;
+    }
```

Comment:
As we are never really sure what we are going to get from user land code... 

```suggestion
    const currentTheme = localStorage.getItem("tr-theme") || "light";
    switch (currentTheme) {
      case "light":
         darkCss.disabled = true;
      case "dark":
      default:
         darkCss.disabled = false;
    }
```